docs/admin/runai-setup/authentication/sso.md

```diff
@@ -41,34 +41,45 @@ You can configure your IdP to map several IdP attributes:
 
-For `Saml 2`:
+=== "SAML 2"
 
-   1. In the `Metadata XML Url` field, enter the URL to the XML Metadata file.
-   2. Find your identity provider's attribute names for `GID`, `GROUPS`, `SUPPLEMENTARYGROUPS` and `UID`. If they are not in line with the Run:ai defaults described in the table above, you can change them here.   
-   3. In the `Logout uri` field, enter the desired URL logout page. If left empty, you will be redirected to the Run:ai portal.
-   4.  Press `Save`.
+    1. Choose `From computer` or `From URL`.
+    
+    	1. For `From computer`, press the `Metadata XML file` field, then select your file for upload. 
+    	2. For `From URL`, in the `Metadata XML Url` field, enter the URL to the XML Metadata file.
+    
+    2. Copy the `Redirect URL` and `Entity ID` for use with your identity provider if required.
+    3. In the `User attributes` field enter the attribute and the value in the identity provider. (optional)
+    4. When complete, press `Save`.
+    5. In the `Logout uri` field, enter the desired URL logout page. If left empty, you will be redirected to the Run:ai portal.
+    6. In the `Session timeout` field, enter the amount of idle time before users are automatically logged out. (Default is 60 minutes)
 
-For `Open ID Connect`:
+    After you have configured the SAML 2 settings, you can download the XML file, and view the identity provider settings. 
+	
+	Press `Download` to download the file.
 
-:octicons-versions-24: Version 2.10 and later.
+	Pres `Edit` to both download the file, and view the:
 
-   1. In the `Discovery Document URL` field, enter the URL to the discovery document.
-   2. In the `Client ID` field, enter the client ID.
-   3. In the `Client Secret` field, enter the client secret.
-   4. Find your identity provider's attribute names for `GID`, `GROUPS`, `SUPPLEMENTARYGROUPS` and `UID`. If they are not in line with the Run:ai defaults described in the table above, you can change them here.   
-   5. In the `Logout uri` field, enter the desired URL logout page. If left empty, you will be redirected to the Run:ai portal.
-   6.  Press `Save`.
+    * Identity provider URL.
+    * Identity provider entity ID.
+    * Certificate expiration date.
 
-Once you press `Save` you will receive a `Redirect URI` and an `Entity ID`. Both values must be set on the IdP side.
+=== "Open ID Connect"
+
+    1. In the `Discovery URL` field, enter the discovery URL .
+    2. In the `Client ID` field, enter the client ID.
+    3. In the `Client Secret` field, enter the client secret.
+    4. In the `User attributes` field enter the attribute and the value in the identity provider. (optional)
+    5.When complete, press `Save`.
+    5. In the `Logout uri` field, enter the desired URL logout page. If left empty, you will be redirected to the Run:ai portal.
+    6. In the `Session timeout` field, enter the amount of idle time before users are automatically logged out. (Default is 60 minutes)
```

Comment:
@jasonnovichRunAI I think this should go separatly because it is not under the identity provider / OIDC